code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -92,6 +92,9 @@
 	id = "goofzombies"
 	limbs_id = "zombie" //They look like zombies
 	sexes = FALSE
+	species_traits = list(HAS_FLESH, HAS_BONE, AGENDER)
+	inherent_traits = list(TRAIT_EASILY_WOUNDED) //you have no skin
```

Comment:
this was not present before and im inclined to say this is not a part of the bugfix, but otherwise i approve